pal/screen-adapter/web/screen-adapter.ts

```diff
@@ -1,11 +1,46 @@
 import { EDITOR, TEST } from 'internal:constants';
 import { SafeAreaEdge } from 'pal/screen-adapter';
 import { systemInfo } from 'pal/system-info';
+import { warnID } from '../../../cocos/core/platform/debug';
 import { EventTarget } from '../../../cocos/core/event/event-target';
 import { Size } from '../../../cocos/core/math';
-import { OS } from '../../system-info/enum-type';
 import { Orientation } from '../enum-type';
 
+const orientationMap = {
```

Comment:
Naming: `OrientationMap` for const enum like type

Reply:
orientationMap is a mapping object, is not used as an enum like type